lib/cache/fetchAndExtract.js

```diff
@@ -29,6 +29,7 @@ module.exports = async ({ url, name, tmpLoc, outLoc, raw, requestOpts }) => {
             reporter.debug(`[CACHE]   to ${chalk.bold(outLoc)}`)
             reporter.debug(`[CACHE]   tmpLoc ${chalk.bold(tmpLoc)}`)
             await mkdirp(tmpLoc)
+            await mkdirp(outLoc)
```

Comment:
*This one* I think needs to be outside the `if` block and probably with a `path.dirname` call as well.

Reply:
See https://github.com/dhis2/cli-helpers-engine/pull/16#issuecomment-497342224.